inference-engine/cmake/compile_options.cmake

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+function(enable_warnings_as_errors TARGET_NAME)
```

Comment:
1. seems to be unused
2. it's supposed to be used in samples code. BUT: this file is not a part of OpenVINO package, so we will have cmake error if `enable_warnings_as_errors` is called from samples

Reply:
remove this file

Reply:
Removed.